src/styles/global.css

```
@@ -59,7 +61,7 @@ h1 {
   background-color: #ff9776;
 }
 
-.expanded {
+.hamburger[aria-expanded="true"] ~ .nav-links {

```

```suggestion
:has(.hamburger[aria-expanded="true"]) .nav-links {
```

To accommodate for the markup changes in `src/components/Header.astro`.